### ISSUE TITLE
compiler message: Error: Could not resolve the package 'flutter_facebook_login' in 'package:flutter_facebook_login/flutter_facebook_login.dart'

### ISSUE THREAD

**Issue by artAst** [roughike/flutter_facebook_login#59]
Hello and good day,

I have tried using this Facebook plugin. It works perfectly on android. Unfortunately, when I build IOS, it is getting this error

```
compiler message: Error: Could not resolve the package 'flutter_facebook_login' in 'package:flutter_facebook_login/flutter_facebook_login.dart'
```

The error runs through FBSDKLoginKit outdated. I am using a FBSDKLoginKit before but Im not sure why it is showing me this error when I used the plugin. Anyways, I tried to do a clean pod install on my machine by:

```
rm -rf ~/Library/Caches/CocoaPods
rm -rf Pods
rm -rf ~/Library/Developer/Xcode/DerivedData/*
pod deintegrate
pod setup
pod install
```
It gets the latest FBSDKLoginKit which is 4.39
But when I run my build, it goes back to 4.28
Im not even sure if this is related issue.

```
compiler message: Error: Could not resolve the package 'flutter_facebook_login' in 'package:flutter_facebook_login/flutter_facebook_login.dart'.
build	11-Dec-2018 20:38:37	    compiler message: package:flutter_facebook_login/flutter_facebook_login.dart: Error: Not found: package:flutter_facebook_login/flutter_facebook_login.dart.
build	11-Dec-2018 20:38:37	    Compiler terminated unexpectedly.
build	11-Dec-2018 20:38:37	    cp: build/aot/App.framework: No such file or directory
```

Please enlighten me on this.
Thanks

**Comment by roughike** [comment]
I haven't encountered this one before, but I think it might be fixed now that I updated the Facebook Login SDK to the latest version. If not, feel free to comment and I'll reopen.

**Comment by markzero** [comment]
I'm having the same issue:

```
[!] CocoaPods could not find compatible versions for pod "FBSDKLoginKit":
  In snapshot (Podfile.lock):
    FBSDKLoginKit (= 4.35.0)

  In Podfile:
    FBSDKLoginKit

    flutter_facebook_login (from `.symlinks/plugins/flutter_facebook_login/ios`) was resolved to 0.0.1, which depends on
      FBSDKLoginKit (= 4.39.1)

Specs satisfying the `FBSDKLoginKit, FBSDKLoginKit (= 4.35.0), FBSDKLoginKit (= 4.39.1)` dependency were found, but they required a higher minimum deployment target.
```
It's a clean install, not sure what to do here...
